ratatui-widgets/src/table.rs

```diff
@@ -863,11 +905,17 @@ impl Table<'_> {
                 buf.set_style(selection_area, row.style);
                 (&self.highlight_symbol).render(selection_area, buf);
             }
-            for ((x, width), cell) in columns_widths.iter().zip(row.cells.iter()) {
-                cell.render(
-                    Rect::new(row_area.x + x, row_area.y, *width, row_area.height),
-                    buf,
-                );
+            let mut column_widths_iter = columns_widths.iter();
+            for current_cell in &row.cells {
+                match Self::get_widths_for_cell(&mut column_widths_iter, current_cell) {
+                    None => {}
```

Comment:
We could do `if let Some(_)` instead of `match` statement. (this would get rid of empty match arm here.)

Reply:
Done